layouts/_default/taxonomy.html

```diff
@@ -0,0 +1,39 @@
+{{ define "title" }}
+  {{- i18n .Data.Singular | title -}}
+  {{- print ": " -}}
+
+  {{- .Title }} · {{ .Site.Title -}}
+{{ end }}
+
+{{ define "content" }}
+  <section class="container list">
+    <h1>{{ i18n "taxonomytext" . }}</h1>
+    <ul>
+      {{ range .Pages.GroupBy "Section" }}
+        <h2>{{ i18n .Key | title }}</h2>
+        
+        {{ if eq .Key "posts" }}
+          <ul>
+            {{ range .Pages }}
+              <li>
+                <span class="date">{{ .Date.Format (.Site.Params.dateFormat | default "January 2, 2006" ) }}</span>
+                <a class="title" href="{{ .Params.ExternalLink | default .RelPermalink }}">{{ .Title }}</a>
+              </li>
+            {{ end }}
+          </ul>
+        {{ else }}
+          <ul class="card-list">
+            {{ range .Pages }}
+              <li>
+                <div class="card-small">
+                  <a class="title" href="{{ .Params.ExternalLink | default .RelPermalink }}">{{ .Title }}</a>
+                  {{ with .Page.Params.dates }}<p>{{ . }}</p>{{ end }}
+                </div>
+              </li>
+            {{ end }}
+          </ul>
+        {{ end }}
+      {{ end }}
+    </ul>
+  </section>
+{{ end }}
```

Comment:
I prefer to go with the default taxonomy and term pages. I don't see any good reason to display projects as cards here too.

Reply:
I wanted to separate the two kinds of contents. I added the number of entries for each term and removed the date because there is no reason to add it.
But I agree, maybe the display of the projects as a card is too much.

Reply:
It is interesting to separate them. But we should use the same listing style for both.
Maybe it is a good idea to make this change generic to any kind of archetype. So, if we add another kind in the future, it will be listed with the same style and group by kind too.

Reply:
> It is interesting to separate them. But we should use the same listing style for both.

How ? You said earlier that `dates` are not applicable to projects.

> Maybe it is a good idea to make this change generic to any kind of archetype. So, if we add another kind in the future, it will be listed with the same style and group by kind too.

To do that, I think I just have to remove `{{ if eq .Key "posts" }}` but again, what if `dates` are not applicable to a future archetype ?

Reply:
Maybe we could verify is the date fields exist and not display it only if it does.

Reply:
Ok, I'm going to do that later today.

Reply:
Hi @luizdepra, how do you want the link (content title) to be displayed ?

Like this ?
![image](https://user-images.githubusercontent.com/35581688/82888659-e7496f80-9f49-11ea-90e3-487cecfbedaf.png)

Or like this ?
![image](https://user-images.githubusercontent.com/35581688/82889015-68086b80-9f4a-11ea-88a7-10535ba79f55.png)

Reply:
Second option. But, is it left-aligned? It doesn't seem so.

Reply:
No, but if I left-align the `.title` element (Hugo Coder), I have the same result as the first option.
If I add a margin-left to this element, it also affects the posts titles.